20231019/context.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+        "id": "@id",
+        "type": "@type",
+        "@vocab": "http://schema.org/",
+        "skos": "http://www.w3.org/2004/02/skos/core#",
+        "prefLabel": {
+            "@id": "skos:prefLabel",
+            "@container": "@language"
+        },
+        "inScheme": "skos:inScheme",
```

Comment:
`inScheme` entfernen, da es auch in der Spec nicht mehr erwähnt wird?

Reply:
Im Kontext sollten wir es nicht entfernen, weil zumindest OERSI die Property benutzt und auf den AMB-Kontext verlinkt, siehe z.B. https://oersi.org/resources/aHR0cHM6Ly93d3cub3JjYS5ucncvY29udGVudC9jZTcwYzFkYS04NDNjLTQyZTItODdjMy1hZWU0NmE1NjEyZDI=?format=json.